scout/server/blueprints/managed_variants/templates/managed_variants/managed_variants.html

```diff
@@ -282,6 +284,14 @@ <h5>
 <script src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-select/1.14.0-beta3/js/bootstrap-select.min.js" integrity="sha512-yrOmjPdp8qH8hgLfWpSFhC/+R9Cj9USL8uJxYIveJZGAiedxyIxwNw4RsLDlcjNlIRR4kkHaDHSmNHAkxFTmgg==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
 <script>
 
+  function removeVariant(variantId) {
+    fetch(`/managed_variant/${variantId}/remove`, {
+        method: "POST",
+    }).finally(() => {
+        document.getElementById("filters_form").submit();
```

Comment:
I found this fix: after removing the managed var, then resubmit the form, with whatever it included previously, even an empty one